labs.yml

```diff
@@ -244,3 +244,9 @@ commands:
         description: workspace_ids which should join a collection. provide a comma separated list of workspace ids.
       - name: target-workspace-id
         description: (Optional) id of a workspace in the target collection. If not specified, ucx will prompt to select from a list
+
+  - name: create-federated-catalog
+    description: (EXPERIMENTAL) Create a federated catalog in the workspace
+
+  - name: enable-hms-federation
+    description: (EXPERIMENTAL) Enable HMS federation based migration flow. When this is enabled, UCX will create a federated HMS catalog which syncs from the workspace HMS.
```

Comment:
When will ucx create the federated HMS catalog?

Reply:
Sorry for the late response. This PR should be on feat/hms-fed branch instead of main. I changed the base now.

The federated catalog is created [here](https://github.com/databrickslabs/ucx/blob/fcdc75a33990c98a2ec3dd1912feeca0a332835e/src/databricks/labs/ucx/hive_metastore/federation.py#L36) which is called from

`create_federated_catalog` [here](https://github.com/databrickslabs/ucx/blob/feat/hms-fed/src/databricks/labs/ucx/cli.py#L506)